packages/terra-text/src/terra-dev-site/doc/example/TextColors.jsx

```diff
@@ -3,10 +3,35 @@ import Text from 'terra-text';
 import classNames from 'classnames/bind';
 import styles from './colors.module.scss';
 
+// :root {
+//   --my-app-text-attention-color: rgb(220, 20, 60);
+//   --my-app-text-success-color: rgb(46, 125, 50);
+//   --my-app-text-info-color: #304ffe;
+//   --my-app-text-default-color: #333;
+// }
+
+// .default {
+//   color: var(--my-app-text-default-color, #333);
+// }
+
+// .attention {
+//   color: var(--my-app-text-attention-color, rgb(220, 20, 60));
+// }
+
+// .success {
+//   color: var(--my-app-text-success-color, rgb(46, 125, 50));
+// }
+
+// .info {
+//   color: var(--my-app-text-info-color, #304ffe);
+// }
+
```

Comment:
CSS in JS taken to a whole new level. Reminder to remove these comments.

Reply:
I had the same question posted here https://github.com/cerner/terra-core/pull/2863#issuecomment-591803668. 
What I think is we can leave `CSS` in the example itself until the toggle CSS functionality is introduced.

Reply:
I don't think commented out CSS in the JS file will be intuitive to users viewing the example code. At a minimum additional documentation should be added describing what the intention of commented out code is.

Alternatively I would recommend creating an additional section in either the existing mdx file or a new mdx file that expands upon implementing custom colors for the text component.

Reply:
I agree with @StephenEsser this isn't the best way to show this functionality to users. I vote for creating a section in the docs that explains how to implement custom colors. Code blocks could be used to make it more readable. While it wouldn't be ideal,  I think it's more clear than commented code in the example jsx file.